intro-HPC/examples/OpenFOAM/OpenFOAM_damBreak.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#PBS -l walltime=1:0:0
+#PBS -l nodes=1:ppn=4
+# check for more recent OpenFOAM modules with 'module avail OpenFOAM'
+module load OpenFOAM/4.1-intel-2017a
```

Comment:
also update this

maybe it's worth defining a variable for this once, can that be done in this chapter?

Reply:
No, it's not possible to define a variable in the chapter and use it here, since LaTeX loads this file literally and doesn't execute any code in it.

I would also be kinda pointless since we only describe how to load this module twice: here and in the beginning of the chapter.